Misc/NEWS.d/next/Core and Builtins/2017-11-12-11-44-22.bpo-28180.HQX000.rst

```diff
@@ -0,0 +1,3 @@
+The platform dependent handling of setting the locale from the environment is
```

Comment:
It's probably clearer to mention the new internal API by name. Something like

> A new internal `_Py_SetLocaleFromEnv(category)` helper function has been added in order
> to improve the consistency of behaviour across different `libc` implementations (e.g. Android
> doesn't support setting the locale from the environment by default)

Reply:
Thanks Nick :-)